docs/src/modules/components/Examples/core-examples.ts

```diff
@@ -84,12 +84,15 @@ export default function examples() {
         'https://codesandbox.io/s/github/mui/toolpad/tree/master/examples/core/auth-vite',
     },
     {
-      title: 'Custom theme with Auth.js and Next.js App router',
+      title: 'Functional Dashboard',
       description:
-        'This app shows you how to get started using Toolpad Core with Next.js, Auth.js and customize the Material UI theme',
-      src: '/static/toolpad/docs/core/auth-next-themed.png',
-      srcDark: '/static/toolpad/docs/core/auth-next-themed-dark.png',
+        'This example shows you how to get started building a dashboard with Toolpad Core, Next.js app router, Auth.js and Material UI components in a customized theme',
```

Comment:
Leading dot:

```suggestion
        'This example shows you how to get started building a dashboard with Toolpad Core, Next.js app router, Auth.js and Material UI components in a customized theme.',
```

Fixed in #4558